packages/nimble-components/src/table-column/extensions/fractional-width-column.ts

```diff
@@ -0,0 +1,46 @@
+import { attr, nullableNumberConverter } from '@microsoft/fast-element';
+import type { TableColumn } from '../base';
+
+export interface FractionalWidthColumn {
+    fractionalWidth: number;
+    disableResize: boolean;
```

Comment:
Thought about removing this `disableResize` property but kept it for now / to get feedback. It's in the HLD but currently unused. Might make sense to remove it until it's needed (presumably once we have interactive column resizing)

Reply:
Oops. I agree. We should remove it.